settings.gradle.kts

```diff
@@ -77,8 +77,12 @@ project(":spark-connector:spark-runtime-3.5").projectDir = file("spark-connector
 include("web:web", "web:integration-test")
 include("docs")
 include("integration-test-common")
+include(":bundles:aws-hadoop-bundle")
 include(":bundles:aws-bundle")
```

Comment:
Can you group them together like `include(":bundles:aws-hadoop-bundle", ":bundles:aws-bundle")`